app/src/main/res/layout/student_item.xml

```diff
@@ -17,15 +17,21 @@
         android:layout_height="70dp">
         <TextView
             android:id="@+id/tvNameSudent"
+            android:textSize="25dp"
```

Comment:
textsize use sp instead of dp 
https://stackoverflow.com/questions/2025282/what-is-the-difference-between-px-dip-dp-and-sp?rq=1

Reply:
and don't hardcode string, dimension, color in code like this
Use strings.xml, colors.xml, dimensions.xml